src/v/cluster/topic_table.cc

```diff
@@ -62,9 +62,29 @@ const model::topic& topic_table::topic_metadata::get_source_topic() const {
 }
 const topic_configuration_assignment&
 topic_table::topic_metadata::get_configuration() const {
-    vassert(
-      is_topic_replicable(),
-      "Query for configuration on a non-replicable topic");
+    static constexpr auto has_null_raft_group_id =
+      [](const partition_assignment& pa) {
+          return pa.group() == null_raft_group_id;
+      };
```

Comment:
why is this `static constexpr`?

Reply:
I had no other reason other then because it can be, opinions?

Reply:
has been removed